### DISCUSSION TITLE
Welcome to ForumFFF Discussions!

### DISCUSSION THREAD

**Discussion by Pomog** [Pomog/ForumFFF#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pomog** [comment]
So what is your plan Denys?

**Comment by TartuDen** [comment]
I finished Login / register form. HTML has its own "validation" of email format. I think for this pet project it will do it. We don't need to focus and add more input valid functionality, at least for now. What will be our next step?

**Reply by Pomog** [reply]
Can you create separate pages for login and registration?

Today I made many corrections to align our code with Trevor Sawler's recommendations.

**Comment by TartuDen** [comment]
the whole project requires some formatting and re-factoring, we should plan it and make it done. Also more unit tests would be great to have.
